Morphic/Morphic/MorphicBar/MorphicBarViewController.swift

```diff
@@ -28,6 +28,8 @@ import MorphicSettings
 /// The View Controller for a MorphicBar showing a collection of actions the user can take
 public class MorphicBarViewController: NSViewController {
     
+    @IBOutlet weak var advancedFeaturesButton: LogoButton!
```

Comment:
Isn't this already an outlet called `logoButton` below?  I'm fine with a name change if you want, but no need for a duplicate outlet

Reply:
Good catch.  I have removed the duplicate outlet in commit # 1bcf328.